Dockerfile

```diff
@@ -20,29 +42,55 @@ RUN apk update && \
         proj-util \
         proj-dev \
         make
+# Setup backend Python dependencies
+COPY --from=extract-deps \
+    /opt/python/requirements.txt /opt/python/
+RUN pip install --user --no-warn-script-location \
+    --no-cache-dir -r /opt/python/requirements.txt
 
-COPY requirements.txt .
 
-RUN pip install \
-    --prefix=/install \
-    --no-cache-dir \
-    --no-warn-script-location \
-    -r requirements.txt
 
+FROM base as runtime
+ARG PYTHON_IMG_TAG
+WORKDIR /usr/src/app
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    PYTHONFAULTHANDLER=1 \
+    PATH="/home/appuser/.local/bin:$PATH" \
+    PYTHON_LIB="/home/appuser/.local/lib/python$PYTHON_IMG_TAG/site-packages" \
```

Comment:
Does `/home/appuser` already exist?

Reply:
It is created below here: https://github.com/hotosm/tasking-manager/blob/7c6e2dc3a116a381cac3833b8d730cf479bf22ed/Dockerfile#L76

I do it in this order for caching reasons / image layering.